### PR TITLE
Fix coding standards

### PR DESCRIPTION
**Pull request by cs-shadow**

### CODE REVIEW

File: src/EntityHelperTrait.php [hunk 1]
```diff
@@ -26,7 +27,7 @@
    *
    * @var \Drupal\Core\Entity\EntityManagerInterface
    */
-  protected $entityManager;
+  protected $entity_manager;
```
Comment (davereid) [inline src/EntityHelperTrait.php]: Not sure this is actually necessary?

File: src/Annotation/EntityEmbedDisplay.php [hunk 2]
```diff
@@ -42,6 +42,6 @@ class EntityEmbedDisplay extends Plugin {
    *
    * @var bool|array
    */
-  public $entity_types = FALSE;
+  public $entityTypes = FALSE;
```
Comment (davereid) [inline src/Annotation/EntityEmbedDisplay.php]: The key used in the annotation is actually 'entity_types' so we can't just change this without also changing the annotations. I think we'll probably want to leave this one alone.

File: src/EntityHelperTrait.php (hunk 1 above)
Comment (cs-shadow) [inline src/EntityHelperTrait.php]: `29 | ERROR   | Variable "entityManager" is camel caps format. do not use 
        |                 | mixed case (camelCase), use lower case and _`

This is code-sniffer output, so I thought it might be a good idea to change to lower case

Comment (davereid) [inline src/EntityHelperTrait.php]: I thnk it's fine to ignore. Core definitely uses way more camel caps format for variables now with D8, so I think it's more of a "code-sniffer would need to be updated for D8 standards" situation.

Comment (cs-shadow) [inline src/EntityHelperTrait.php]: Oh, okay. Reverted this change.

File: src/Plugin/Derivative/FieldFormatterDeriver.php [hunk 3]
```diff
@@ -24,16 +24,16 @@ class FieldFormatterDeriver extends DerivativeBase implements ContainerDerivativ
    *
    * @var \Drupal\Core\Field\FormatterPluginManager.
    */
-  protected $formatterManager;
+  protected $formatter_manager;
```
Comment (davereid) [inline src/Plugin/Derivative/FieldFormatterDeriver.php]: Let's revert this change as well.
